add epicsbase to feedstock

jennmald opened the pull request:
<!--
Thank you for your pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

jennmald committed:
MNT: Re-rendered with conda-smithy 3.53.3 and conda-forge-pinning 2025.12.03.14.16.13

Other tools:
- conda-build 25.11.0
- rattler-build 0.53.0
- rattler-build-conda-compat 1.4.10

jennmald committed:
remove setup dso and add epicsbase

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/meta.yaml```) and found it was in an excellent condition.

I do have some suggestions for making it better though...


For **recipe/meta.yaml**:

 * ℹ️ PyPI default URL is now pypi.org, and not pypi.io. You may want to update the default source url.


<sub>This message was generated by GitHub Actions workflow run [https://github.com/conda-forge/conda-forge-webservices/actions/runs/20180173809](https://github.com/conda-forge/conda-forge-webservices/actions/runs/20180173809). Examine the logs at this URL for more detail.</sub>

jennmald committed:
MNT: Re-rendered with conda-smithy 3.53.3 and conda-forge-pinning 2025.12.04.13.15.23

Other tools:
- conda-build 25.11.0
- rattler-build 0.53.0
- rattler-build-conda-compat 1.4.10

jennmald committed:
add patch

jennmald committed:
remove comments and fix patch
